accounts-db/src/accounts.rs

```diff
@@ -1933,7 +1933,7 @@ mod tests {
                 assert_eq!(loaded_transaction.program_indices.len(), 1);
                 assert_eq!(loaded_transaction.program_indices[0].len(), 0);
             }
-            (Err(e), _nonce) => Err(e).unwrap(),
+            (Err(e), _nonce) => panic!("{e:?}"),
```

Comment:
I don't think these panics need debug formatting:
```suggestion
            (Err(e), _nonce) => panic!("{e}"),
```

Reply:
hrm, why was i thinking panic tosses `Debug`...

Reply:
switched to `Display` in ee64385408ee72e6a0ec705bbdd18f62df7ac645